.github/workflows/build.yml

```diff
@@ -100,11 +100,12 @@ jobs:
       - name: Build nrf91 firmware
         working-directory: thingy91x-oob/app
         run: |
-          echo "CONFIG_MEMFAULT_NCS_PROJECT_KEY=\"${{ secrets.MEMFAULT_PROJECT_KEY }}\"" >> overlay-memfault.conf
```

Comment:
I wanted to avoid re-using the modified config in the debug build. So copy the original config to a different name before modifying it.